pico/tools/src/main/java/io/helidon/pico/tools/ExternalModuleCreatorDefault.java

```diff
@@ -104,11 +103,11 @@ public ExternalModuleCreatorResponse prepareToCreateExternalModule(ExternalModul
             // process each found service type
             scan.get().getAllStandardClasses()
                     .stream()
-                    .filter((classInfo) -> packageNames.contains(classInfo.getPackageName()))
-                    .filter((classInfo) -> !classInfo.isInterface())
-                    .filter((classInfo) -> !classInfo.isExternalClass())
-                    .filter((classInfo) -> !isPrivate(classInfo.getModifiers()))
-                    .filter((classInfo) -> !classInfo.isInnerClass() || req.innerClassesProcessed())
+                    .filter(classInfo -> packageNames.contains(classInfo.getPackageName()))
```

Comment:
This could be done in a bit more readable way, such as:
- Predicate.not() can be statically imported
```
            scan.get()
                    .getAllStandardClasses()
                    .stream()
                    .filter(this::includedPackage)
                    .filter(Predicate.not(ClassInfo::isInterface))
                    .filter(Predicate.not(ClassInfo::isExternalClass))
                    .filter(Predicate.not(ClassInfo::isPrivate))
                    .filter(this::notInnerOrProcessed)
                    .forEach(this::processServiceType);
```